tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -1204,6 +1206,12 @@ static __always_inline int save_file_path_to_str_buf(buf_t *string_p, struct fil
     return buf_off;
 }
 
+static __always_inline int save_file_path_to_str_buf(buf_t *string_p, struct file* file)
+{
+    struct path f_path = get_path_from_file(file);
+    return save_path_to_str_buf(string_p, &f_path);
```

Comment:
We read struct path both here and in line 1138.
Maybe it is possible to do just one read by using &file->f_path and removing the previous line
(It is possible that the verifier won't like it)